### PR TITLE
fix 204 issue

### PR DESCRIPTION
**Pull request by chris110408**
#### Issue addressed

fixes #

### CODE REVIEW

**Comment by mattwr18** [comment]
@chris110408 can you please run `yarn fix` to fix the linting issues?
it's failing the build cause of them... then I can give a review :smile_cat:

**Review by joaopapereira — CHANGES_REQUESTED** [review]
Looks good, please fix the linting and we can merge the PR

**Comment by chris110408** [comment]
@mattwr18 @joaopapereira  the lint has been fixed, sorry for not checking the lint before pull request

File: cypress/integration/common/web_steps.js [hunk 1]
```diff
@@ -299,6 +299,8 @@ Then('I should be able to create a new project', () => {
       .type('http://i.imgur.com/WPQcOUl.png')
       .get('textarea[name=description]')
       .type('A new project')
+      .get('textarea[name= slack_channel_name]')
+      .type('A new project')
```
Comment (joaopapereira) [inline cypress/integration/common/web_steps.js]:
```suggestion
      .type('slack_channel')
```

**Review by joaopapereira — CHANGES_REQUESTED** [review]
Sorry just realized there is an incorrect text in the fix. Can you please fix it?

File: cypress/integration/common/web_steps.js (hunk 1 above)
Comment (chris110408) [reply cypress/integration/common/web_steps.js]: @joaopapereira  the text has been fixed

File: src/components/ProjectForm.js [hunk 2]
```diff
@@ -27,6 +27,10 @@ export const ProjectForm = props => {
         label='Description'
         placeholder='Description'
       />
+      <Field name='slack_channel_name' component={TextAreaField}
+        label='slack channel name'
```
Comment (mattwr18) [inline src/components/ProjectForm.js]: I wonder if it's not better to capitalize the label?

**Review by mattwr18 — COMMENTED** [review]
Looks good, see my comment :point_down:

File: src/components/ProjectForm.js (hunk 2 above)
Comment (chris110408) [reply src/components/ProjectForm.js]: @mattwr18 fixed

**Review by joaopapereira — APPROVED** [review]

**Pull request merged**